content/en/continuous_testing/cicd_integrations/configuration.md

```diff
@@ -85,11 +87,24 @@ In the global configuration file, you can configure the following options:
 `datadogSite`
 : The Datadog instance to which request is sent. The default is `datadoghq.com`. Your Datadog site is {{< region-param key="dd_site" code="true" >}}.
 
+`failOnCriticalErrors`
+: A boolean flag that will fail the CI job if no tests were triggered, or results couldn't be fetched from Datadog. The default is set to `false`.
+
+`failOnMissingTests`
+: A boolean flag that will fail the CI job if at least one test is missing within a run (for example, if it has been removed/deleted). The default is set to `false`.
+
+`failOnTimeout`
+: A boolean flag that will fail the CI job if at least one test exceees the default test timeout. The default is set to `true`.
+
 `files`
 : Glob pattern to detect Synthetic tests config files.
 
 `global`
-: Overrides of Synthetic tests applied to all tests ([see below for descriptions of each field](#configure-tests)).
+: Overrides of Synthetic tests applied to all tests ([see below for descriptions of each field](#additional-configuration)).
```

Comment:
Which header does this anchor link to?

```suggestion
: Overrides of Synthetic tests applied to all tests. [See below for descriptions of each field]().
```

Reply:
@alai97 Do we want to remove the link to the `additional configuration` section?